app/src/test/resources/io/apicurio/registry/content/xml-schema-expected.xsd

```diff
@@ -0,0 +1,20 @@
+<xs:schema xmlns:xs="http://www.w3.org/2001/XMLSchema">
+<xs:element name="shiporder">
+  <xs:complexType>
+  <xs:sequence>
+	<xs:element name="orderperson" type="xs:string"></xs:element>
+	 <xs:element name="shipto">
+	   <xs:complexType>
+		<xs:sequence>
+		   <xs:element name="name" type="xs:string"></xs:element>
+		   <xs:element name="address" type="xs:string"></xs:element>
+		   <xs:element name="city" type="xs:string"></xs:element>
+		   <xs:element name="country" type="xs:string"></xs:element>
+	     </xs:sequence>
+	    </xs:complexType>
+	   </xs:element>
+	 </xs:sequence>
+	 <xs:attribute name="orderid" type="xs:string" use="required"></xs:attribute>
+  </xs:complexType>
+</xs:element>
+</xs:schema>
```

Comment:
What additional options are there for the XML `Canonicalizer`?  Looking at this file I am noticing the strange/inconsistent indentation for some of the elements.  For example lines 8 and 13 don't line up, and in general the tabbing seems to be preserved from the source.  I'm wondering if there is a more aggressive option we can enable for XML canonicalization that normalizes the indentation of the elements, removing e.g. extra whitespace.

Reply:
@EricWittmann I could be wrong but it doesn't look like there are any options to do that..  I've been looking into it and I think the spec says to normalise the whitespace within tags etc but not outside this..  ```Whitespace outside of the document element and within start and end tags is normalized.
All whitespace in character content is retained (excluding characters removed during line feed normalization)```  I think that means the indentation should be preserved. WDYT?

Reply:
Yeah if that's the language in the spec then it's doing what the spec says.  Do you have a link to that language?

Reply:
Yep - https://www.w3.org/TR/2001/REC-xml-c14n-20010315#Terminology 1.1